core/listings.go

```diff
@@ -691,6 +691,17 @@ func validateListing(listing *pb.Listing) (err error) {
 	if len(listing.Metadata.Language) > WordMaxCharacters {
 		return fmt.Errorf("Language is longer than the max of %d characters", WordMaxCharacters)
 	}
+	if len(listing.Metadata.AcceptedCurrencies) == 0 {
+		return errors.New("At least one accepted currency must be provided")
```

Comment:
Making errors like this named (i.e. package-level vars) would be super cool but not required.